apps/src/gamelab/spritelab/spriteCommands.js

```diff
@@ -0,0 +1,50 @@
+import * as spriteUtils from './spriteUtils.js';
+
+export const commands = {
+  makeSprite(animation, location) {
+    var sprite = this.createSprite(location.x, location.y);
+    spriteUtils.nativeSpriteMap[spriteUtils.spriteId] = sprite;
+    sprite.id = spriteUtils.spriteId;
+    if (animation) {
+      sprite.setAnimation(animation);
+    }
+    spriteUtils.spriteId++;
+    return sprite.id;
+  },
+
+  setAnimation(spriteIndex, animation) {
+    let sprites = spriteUtils.singleOrGroup(spriteIndex);
+    if (sprites) {
+      sprites.forEach(sprite => {
+        sprite.setAnimation(animation);
+      });
+    }
+  },
+
+  getProp(spriteIndex, prop) {
+    if (!spriteIndex) {
+      return undefined;
+    }
+
+    let sprite = spriteUtils.singleOrGroup(spriteIndex)[0];
+    if (prop === 'scale') {
+      return sprite.getScale() * 100;
+    } else if (prop === 'costume') {
+      return sprite.getAnimationLabel();
+    } else if (prop === 'y') {
+      return 400 - sprite.y;
```

Comment:
Add a comment here saying why we subtract from 400 (or if this is in a const file somewhere, it would be nice to use that)